libeos-payg/service.c

```diff
@@ -526,6 +546,30 @@ provider_notify_enabled_cb (GObject    *object,
   self->holding = enabled;
 }
 
+static void provider_unlocked_cb (EpgProvider *provider,
+                                  gpointer     user_data)
+{
+  EpgService *self = EPG_SERVICE (user_data);
+
+  g_message ("EpgProvider emitted 'unlocked' signal");
+
+  /* Check self->eospayg_active_efivar to determine if we're in backward compat
+   * mode for Phase 2, and to allow EOSPAYG_debug to prevent deletion of
+   * EOSPAYG_active (via EPG_DEBUG_EOSPAYG_ACTIVE_OFF).
+   */
+  if (self->eospayg_active_efivar)
+    {
+      if (efi_del_variable (EOSPAYG_GUID, "EOSPAYG_active") < 0)
+        g_warning ("Failed to delete EOSPAYG_active upon unlock");
+      else
+        g_message ("Deleted EOSPAYG_active upon unlock");
+
+      /* FIXME: Perhaps we should also change the Secure Boot settings so the
```

Comment:
Let's cut this for now just to reduce our testing burden. But I agree we should follow up on it soon so I filed https://phabricator.endlessm.com/T28511 . (I thought we already had a task for this but I couldn't find it)

Reply:
I don't see how a code comment increases testing burden?

Reply:
I added that phabricator link to the comment. I guess you meant our burden would be increased if we implemented that change.